Dockerfile

```
@@ -0,0 +1,59 @@
+FROM ruby:2.6-slim-buster
+
+ENV APP_HOME /opt/app
+ENV BUNDLE_BIN $GEM_HOME/bin
+ENV PATH $BUNDLE_BIN:$PATH
+
+ENV RAILS_SERVE_STATIC_FILES true
+ENV RAILS_ENV production
+
+ENV HOME $APP_HOME
+ENV USER deployer
+ENV PORT 3000
+
+RUN apt update && \
+    apt install -qq -y build-essential git libxml2-dev libsass-dev \
+                       libpq-dev freetds-dev libmariadb-dev unixodbc-dev \
+                       wget nodejs libxml2 libxml2-utils  \
+                       openssl ssh tzdata alien
+
+ENV base_url    "https://download.oracle.com/otn_software/linux/instantclient/193000/"
+ENV basic_rpm   "oracle-instantclient19.3-basic-19.3.0.0.0-1.x86_64.rpm"
+ENV sqlplus_rpm "oracle-instantclient19.3-sqlplus-19.3.0.0.0-1.x86_64.rpm"
+ENV devel_rpm   "oracle-instantclient19.3-devel-19.3.0.0.0-1.x86_64.rpm"
+ENV odbc_rpm    "oracle-instantclient19.3-odbc-19.3.0.0.0-1.x86_64.rpm"
+
+RUN wget $base_url$basic_rpm \
```

Ahhh... me gusta que la / separe las variable, así:
```suggestion
RUN wget $base_url/$basic_rpm \
```